[2024-06-03T17:38:23.772+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-03T17:38:23.787+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: chess_etl.blunders_etl scheduled__2019-09-01T00:00:00+00:00 [queued]>
[2024-06-03T17:38:23.794+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: chess_etl.blunders_etl scheduled__2019-09-01T00:00:00+00:00 [queued]>
[2024-06-03T17:38:23.795+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-06-03T17:38:23.806+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): blunders_etl> on 2019-09-01 00:00:00+00:00
[2024-06-03T17:38:23.811+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=114) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-03T17:38:23.813+0000] {standard_task_runner.py:63} INFO - Started process 117 to run task
[2024-06-03T17:38:23.813+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'chess_etl', 'blunders_etl', 'scheduled__2019-09-01T00:00:00+00:00', '--job-id', '23', '--raw', '--subdir', 'DAGS_FOLDER/chess_dag.py', '--cfg-path', '/tmp/tmpec_3p0yi']
[2024-06-03T17:38:23.814+0000] {standard_task_runner.py:91} INFO - Job 23: Subtask blunders_etl
[2024-06-03T17:38:23.847+0000] {task_command.py:426} INFO - Running <TaskInstance: chess_etl.blunders_etl scheduled__2019-09-01T00:00:00+00:00 [running]> on host a5a4421409ea
[2024-06-03T17:38:23.908+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='chess_etl' AIRFLOW_CTX_TASK_ID='blunders_etl' AIRFLOW_CTX_EXECUTION_DATE='2019-09-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2019-09-01T00:00:00+00:00'
[2024-06-03T17:38:23.909+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-03T17:38:23.910+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/utils/context.py:320: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(k, replacements))

[2024-06-03T17:38:23.917+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-03T17:38:23.923+0000] {taskinstance.py:2905} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 465, in _execute_task
    result = _execute_callable(context=context, **execute_callable_kwargs)
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/taskinstance.py", line 432, in _execute_callable
    return execute_callable(context=context, **execute_callable_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/baseoperator.py", line 400, in wrapper
    return func(self, *args, **kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 235, in execute
    return_value = self.execute_callable()
                   ^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/operators/python.py", line 252, in execute_callable
    return self.python_callable(*self.op_args, **self.op_kwargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/dags/chess_dag.py", line 23, in blunders_etl_task
    return get_monthly_blunders(execution_date)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/opt/airflow/dags/blunder_etl_code.py", line 89, in get_monthly_blunders
    connection = BaseHook.get_connection('chess_project_s3')
                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/hooks/base.py", line 83, in get_connection
    conn = Connection.get_connection_from_secrets(conn_id)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/airflow/.local/lib/python3.12/site-packages/airflow/models/connection.py", line 519, in get_connection_from_secrets
    raise AirflowNotFoundException(f"The conn_id `{conn_id}` isn't defined")
airflow.exceptions.AirflowNotFoundException: The conn_id `chess_project_s3` isn't defined
[2024-06-03T17:38:23.929+0000] {taskinstance.py:1206} INFO - Marking task as FAILED. dag_id=chess_etl, task_id=blunders_etl, run_id=scheduled__2019-09-01T00:00:00+00:00, execution_date=20190901T000000, start_date=20240603T173823, end_date=20240603T173823
[2024-06-03T17:38:23.936+0000] {standard_task_runner.py:110} ERROR - Failed to execute job 23 for task blunders_etl (The conn_id `chess_project_s3` isn't defined; 117)
[2024-06-03T17:38:23.947+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 1
[2024-06-03T17:38:23.957+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-03T17:38:23.959+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2024-06-03T17:45:02.186+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-03T17:45:02.201+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: chess_etl.blunders_etl scheduled__2019-09-01T00:00:00+00:00 [queued]>
[2024-06-03T17:45:02.207+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: chess_etl.blunders_etl scheduled__2019-09-01T00:00:00+00:00 [queued]>
[2024-06-03T17:45:02.207+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-06-03T17:45:02.216+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): blunders_etl> on 2019-09-01 00:00:00+00:00
[2024-06-03T17:45:02.222+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=151) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-03T17:45:02.223+0000] {standard_task_runner.py:63} INFO - Started process 153 to run task
[2024-06-03T17:45:02.223+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'chess_etl', 'blunders_etl', 'scheduled__2019-09-01T00:00:00+00:00', '--job-id', '81', '--raw', '--subdir', 'DAGS_FOLDER/chess_dag.py', '--cfg-path', '/tmp/tmpku2yxeqt']
[2024-06-03T17:45:02.224+0000] {standard_task_runner.py:91} INFO - Job 81: Subtask blunders_etl
[2024-06-03T17:45:02.256+0000] {task_command.py:426} INFO - Running <TaskInstance: chess_etl.blunders_etl scheduled__2019-09-01T00:00:00+00:00 [running]> on host f55fff24b127
[2024-06-03T17:45:02.321+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='chess_etl' AIRFLOW_CTX_TASK_ID='blunders_etl' AIRFLOW_CTX_EXECUTION_DATE='2019-09-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2019-09-01T00:00:00+00:00'
[2024-06-03T17:45:02.323+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-03T17:45:02.324+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/utils/context.py:320: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(k, replacements))

[2024-06-03T17:45:02.331+0000] {base.py:84} INFO - Using connection ID 'chess_project_s3' for task execution.
[2024-06-03T17:45:03.544+0000] {logging_mixin.py:188} INFO - No games for period 2019_09
[2024-06-03T17:45:03.545+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-06-03T17:45:03.545+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-03T17:45:03.551+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=chess_etl, task_id=blunders_etl, run_id=scheduled__2019-09-01T00:00:00+00:00, execution_date=20190901T000000, start_date=20240603T174502, end_date=20240603T174503
[2024-06-03T17:45:03.600+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-03T17:45:03.611+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-03T17:45:03.613+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2024-06-08T14:10:30.562+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-08T14:10:30.577+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: chess_etl.blunders_etl scheduled__2019-09-01T00:00:00+00:00 [queued]>
[2024-06-08T14:10:30.582+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: chess_etl.blunders_etl scheduled__2019-09-01T00:00:00+00:00 [queued]>
[2024-06-08T14:10:30.583+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-06-08T14:10:30.593+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): blunders_etl> on 2019-09-01 00:00:00+00:00
[2024-06-08T14:10:30.598+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=167) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-08T14:10:30.600+0000] {standard_task_runner.py:63} INFO - Started process 169 to run task
[2024-06-08T14:10:30.600+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'chess_etl', 'blunders_etl', 'scheduled__2019-09-01T00:00:00+00:00', '--job-id', '221', '--raw', '--subdir', 'DAGS_FOLDER/chess_dag.py', '--cfg-path', '/tmp/tmpyed0jtt8']
[2024-06-08T14:10:30.601+0000] {standard_task_runner.py:91} INFO - Job 221: Subtask blunders_etl
[2024-06-08T14:10:30.632+0000] {task_command.py:426} INFO - Running <TaskInstance: chess_etl.blunders_etl scheduled__2019-09-01T00:00:00+00:00 [running]> on host 3c9d89bfdaf1
[2024-06-08T14:10:30.694+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='chess_etl' AIRFLOW_CTX_TASK_ID='blunders_etl' AIRFLOW_CTX_EXECUTION_DATE='2019-09-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2019-09-01T00:00:00+00:00'
[2024-06-08T14:10:30.695+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-08T14:10:30.696+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/utils/context.py:320: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(k, replacements))

[2024-06-08T14:10:30.702+0000] {base.py:84} INFO - Using connection ID 'chess_project_s3' for task execution.
[2024-06-08T14:10:31.587+0000] {logging_mixin.py:188} INFO - No games for period 2019_09
[2024-06-08T14:10:31.588+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-06-08T14:10:31.588+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-08T14:10:31.594+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=chess_etl, task_id=blunders_etl, run_id=scheduled__2019-09-01T00:00:00+00:00, execution_date=20190901T000000, start_date=20240608T141030, end_date=20240608T141031
[2024-06-08T14:10:31.616+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-08T14:10:31.627+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-08T14:10:31.629+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2024-06-08T14:51:57.926+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-08T14:51:57.940+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: chess_etl.blunders_etl scheduled__2019-09-01T00:00:00+00:00 [queued]>
[2024-06-08T14:51:57.946+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: chess_etl.blunders_etl scheduled__2019-09-01T00:00:00+00:00 [queued]>
[2024-06-08T14:51:57.946+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-06-08T14:51:57.956+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): blunders_etl> on 2019-09-01 00:00:00+00:00
[2024-06-08T14:51:57.962+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=277) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-08T14:51:57.964+0000] {standard_task_runner.py:63} INFO - Started process 279 to run task
[2024-06-08T14:51:57.963+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'chess_etl', 'blunders_etl', 'scheduled__2019-09-01T00:00:00+00:00', '--job-id', '257', '--raw', '--subdir', 'DAGS_FOLDER/chess_dag.py', '--cfg-path', '/tmp/tmp8jnew94w']
[2024-06-08T14:51:57.964+0000] {standard_task_runner.py:91} INFO - Job 257: Subtask blunders_etl
[2024-06-08T14:51:57.996+0000] {task_command.py:426} INFO - Running <TaskInstance: chess_etl.blunders_etl scheduled__2019-09-01T00:00:00+00:00 [running]> on host bf0ade909435
[2024-06-08T14:51:58.063+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='chess_etl' AIRFLOW_CTX_TASK_ID='blunders_etl' AIRFLOW_CTX_EXECUTION_DATE='2019-09-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2019-09-01T00:00:00+00:00'
[2024-06-08T14:51:58.064+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-08T14:51:58.066+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/utils/context.py:320: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(k, replacements))

[2024-06-08T14:51:58.073+0000] {base.py:84} INFO - Using connection ID 'chess_project_s3' for task execution.
[2024-06-08T14:51:59.122+0000] {logging_mixin.py:188} INFO - No games for period 2019_09
[2024-06-08T14:51:59.123+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-06-08T14:51:59.123+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-08T14:51:59.130+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=chess_etl, task_id=blunders_etl, run_id=scheduled__2019-09-01T00:00:00+00:00, execution_date=20190901T000000, start_date=20240608T145157, end_date=20240608T145159
[2024-06-08T14:51:59.180+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-08T14:51:59.192+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-08T14:51:59.193+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
[2024-07-07T21:20:25.707+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-07-07T21:20:25.723+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: chess_etl.blunders_etl scheduled__2019-09-01T00:00:00+00:00 [queued]>
[2024-07-07T21:20:25.728+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: chess_etl.blunders_etl scheduled__2019-09-01T00:00:00+00:00 [queued]>
[2024-07-07T21:20:25.729+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-07-07T21:20:25.738+0000] {taskinstance.py:2330} INFO - Executing <Task(PythonOperator): blunders_etl> on 2019-09-01 00:00:00+00:00
[2024-07-07T21:20:25.743+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=1318) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-07-07T21:20:25.745+0000] {standard_task_runner.py:63} INFO - Started process 1321 to run task
[2024-07-07T21:20:25.745+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'chess_etl', 'blunders_etl', 'scheduled__2019-09-01T00:00:00+00:00', '--job-id', '588', '--raw', '--subdir', 'DAGS_FOLDER/chess_dag.py', '--cfg-path', '/tmp/tmpmn64i7dh']
[2024-07-07T21:20:25.746+0000] {standard_task_runner.py:91} INFO - Job 588: Subtask blunders_etl
[2024-07-07T21:20:25.778+0000] {task_command.py:426} INFO - Running <TaskInstance: chess_etl.blunders_etl scheduled__2019-09-01T00:00:00+00:00 [running]> on host 6569f5155f0c
[2024-07-07T21:20:25.838+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='chess_etl' AIRFLOW_CTX_TASK_ID='blunders_etl' AIRFLOW_CTX_EXECUTION_DATE='2019-09-01T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2019-09-01T00:00:00+00:00'
[2024-07-07T21:20:25.839+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-07-07T21:20:25.840+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/utils/context.py:320: AirflowContextDeprecationWarning: Accessing 'execution_date' from the template is deprecated and will be removed in a future version. Please use 'data_interval_start' or 'logical_date' instead.
  warnings.warn(_create_deprecation_warning(k, replacements))

[2024-07-07T21:20:25.846+0000] {base.py:84} INFO - Using connection ID 'chess_project_s3' for task execution.
[2024-07-07T21:20:26.764+0000] {logging_mixin.py:188} INFO - No games for period 2019_09
[2024-07-07T21:20:26.765+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-07-07T21:20:26.765+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-07-07T21:20:26.772+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=chess_etl, task_id=blunders_etl, run_id=scheduled__2019-09-01T00:00:00+00:00, execution_date=20190901T000000, start_date=20240707T212025, end_date=20240707T212026
[2024-07-07T21:20:26.801+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-07-07T21:20:26.821+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-07-07T21:20:26.822+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
